github/repos_collaborators.go

```diff
@@ -110,6 +110,19 @@ func (s *RepositoriesService) AddCollaborator(ctx context.Context, owner, repo,
 	return s.client.Do(ctx, req, nil)
 }
 
+// AddCollaboratorDirectly adds the specified GitHub user as a collaborator to the given repo.
+//
```

Comment:
How about adding "without sending an invite"?
Also, maybe note that this will most likely be going away soon... or maybe even mark it "Deprecated" here on creation.